src/containers/OrgDetails.tsx

```diff
@@ -150,7 +150,7 @@ export const OrgDetails = () => {
     defaultValues: defaultValuesSessionStorage || defaultValues
   });
   const { watch, handleSubmit } = methods;
-  const { privacy } = watch();
+  const { privacy, invoice } = watch();
```

Comment:
all the fields that you added conditionaly based on whether the `invoice` has ben checked, have to be required in the `Yup` validation.

So based on whether the `invoice` you need to dynamically change the form validation.